modules/delegates.js

```diff
@@ -315,12 +317,12 @@ __private.checkDelegates = function (publicKey, votes, state, cb) {
 				return setImmediate(cb, 'Invalid public key');
 			}
 
-			modules.accounts.getAccount({ publicKey: publicKey, isDelegate: 1 }, function (err, account) {
+			modules.accounts.getAccount({ publicKey: publicKey }, function (err, account) {
```

Comment:
Use delegates table for that - can be done later

Reply:
yes its planned for later. logic is available but not done here in this PR